packages/theme/css/characteristics/content.css

```diff
@@ -5,6 +5,7 @@
   --salt-content-secondary-foreground: var(--salt-palette-neutral-secondary-foreground);
   --salt-content-secondary-foreground-disabled: var(--salt-palette-neutral-secondary-foreground-disabled);
   --salt-content-accent-foreground: var(--salt-palette-accent-foreground-informative);
+  --salt-content-attention-foreground: var(--salt-palette-error-foreground-informative);
```

Comment:
@origami-z this is my mistake, could we change this to point to `--salt-palette-negative-foreground` instead rather than error-foreground

Reply:
Updated